From 0c6c593783f2d64a429ad38523661a915aa462fc Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Sun, 13 Mar 2022 13:44:47 +0100
Subject: [PATCH 1/3] nss-crypto: fix SHA1 header include

SHA1 header has been merged to the generic SHA one,
and with that the cryptohash.h was dropped.

So, fix include in kernels 5.8 and newer.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 v2.0/src/nss_crypto_hlos.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/v2.0/src/nss_crypto_hlos.h
+++ b/v2.0/src/nss_crypto_hlos.h
@@ -55,7 +55,9 @@
 #include <linux/interrupt.h>
 #include <linux/delay.h>
 #include <linux/vmalloc.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 8, 0)
 #include <linux/cryptohash.h>
+#endif
 #include <crypto/sha.h>
 #include <crypto/aes.h>
 #include <crypto/des.h>
